app/presenters/url.rb

```diff
@@ -0,0 +1,22 @@
+class Url
+  include Rails.application.routes.url_helpers
+  include ActionView::Helpers::UrlHelper
+
+  attr_reader :link_text, :path_name, :params
+
+  def initialize(link_text:, path_name:, params: {})
+    @link_text = link_text
+    @path_name = path_name
+    @params = params
+  end
+
+  def to_s
```

Comment:
In the same vein as the class having a confusing name to me, `Url.to_s` is returning HTML for a link, which is the opposite of wha tI would expect, what if we made this `as_link` or `a_tag` or `html_link`? Anything but plain `to_s` please!